src/main/webapp/WEB-INF/views/admin/project/bomCompare-js.jsp

```diff
@@ -7,7 +7,11 @@
 		$("#beforePrjId").val("${beforePrjId}");
 		$("#afterPrjId").val("${afterPrjId}");
 
-		'use strict';
+        if (beforePrjId !== "0000" && afterPrjId !== "0000") {
+            bomCompareList.search();
+        }
+
+        'use strict';
```

Comment:
@dener8 This code should be indented because it is contained within the scope of the anonymous function.

```suggestion
                if (beforePrjId !== "0000" && afterPrjId !== "0000") {
                    bomCompareList.search();
                }

                'use strict';
```